lib/beaker/host/windows/pkg.rb

```diff
@@ -25,15 +22,6 @@ def install_package(name, cmdline_args = '')
       cygwin = "setup-x86.exe"
     end
 
-    if not check_for_command(cygwin)
```

Comment:
I've now added functionality to ensure that cygwin is installed using cygcheck during host validation.